packages/odyssey-storybook/.storybook/components/MuiThemeDecorator.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
+import { ThemeProvider } from "@storybook/theming";
+import { theme } from "@okta/odyssey-react-mui";
+import type { DecoratorFn } from "@storybook/react";
+
+export const MuiThemeDecorator: DecoratorFn = (Story) => (
```

Comment:
maybe add a note that this should be removed with storybook 6.5 https://github.com/storybookjs/storybook/releases/tag/v6.5.0-alpha.31